cmd/constants.go

```diff
@@ -2,18 +2,8 @@
 // See the file LICENSE for licensing terms.
 package cmd
```

Comment:
should be better to put this constants into root.go? seems to be used only there

Reply:
I ended up moving them to the constants package.